ibm_catalog.json

```diff
@@ -122,7 +124,9 @@
             },
             {
               "key": "existing_resource_group_name",
-              "required": true,
+              "display_name": "resource_group",
+              "required": false,
+              "default_value": "Default",
```

Comment:
Do not need to add `required` if its not true. And if default value is defined in variables.tf, it will be picked up from there. No need to add here.